nixos/modules/services/torrent/cross-seed.nix

```diff
@@ -40,6 +49,16 @@ in
       description = "Cross-seed config directory";
     };
 
+    useGenConfigDefaults = mkEnableOption "" // {
```

Comment:
I'd use mk option here instead of mkEnableOption just so it's a bit cleaner.

Other than that, I'd add that other settings will override these, and that it won't merge recursively.

Reply:
Yeah sure.